app/marionette/src/server/framework-preset-marionette.ts

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Configuration } from 'webpack';
+
+export function webpack(config: Configuration) {
+  return config;
+}
```

Comment:
Not sure we need this but I didn't want to make changes to production code other than a simple TS migration.